README.md

```diff
@@ -29,7 +29,7 @@ end
 
 This nicely complements Ruby's built-in `attr_accessor`, `attr_reader` and `attr_writer`.
 
-Supports positional arguments as well as optional and required hash arguments.
+Supports positional arguments as well as optional and required hash arguments with or without default values.
```

Comment:
Maybe ". Hash arguments can have default values." to be clear that it doesn't apply to positional arguments.